iBudget/fund/views.py

```diff
@@ -206,3 +207,89 @@ def create_new_goal(request):
             shared_group=shared_group):
         return HttpResponse(status=201)
     return HttpResponse(status=409)
+
+def create_initial_balance(user, begin_date, start_date, fund_id):
+    """Creating fund balance.
+            Args:
+                user (UserProfile): user.
+                begin_date (date): the beginning of statistic period for this user
+                start_date (date): the beginning of statistic period
+                fund_id (int): fund ID
+            Returns:
+                Balance value
+    """
+    return sum(IncomeHistory.filter_by_fund_id(fund_id).filter(
```

Comment:
Such kind of complex construction in return statement is not very readable. Not a bug but it is better to avoid it.

Reply:
I understand it but there were some problems with pylint (somethig like "too many variables...")